src/components/capture/creation/SchemaEditor.tsx

```diff
@@ -3,12 +3,19 @@ import { DialogContentText, Paper, useTheme } from '@mui/material';
 import * as monacoEditor from 'monaco-editor/esm/vs/editor/editor.api';
 import { useRef } from 'react';
 import { FormattedMessage } from 'react-intl';
+import useSchemaEditorStore, {
+    SchemaEditorState,
+} from '../../../stores/SchemaEditorStore';
```

Comment:
We should add `stores/*` to the tsconfig so we don't have to go up the file path like this.

Reply:
Pending update. Yes, I arrived at that conclusion shortly after opening this PR. It was meant to be a draft to review and correct any mistakes before anyone else took a critical look, but I forget to press that button yesterday evening.